content/applications/essentials/keyboard_shortcuts.rst

```diff
@@ -0,0 +1,93 @@
+==================
+Keyboard shortcuts
+==================
+
+.. |kbs| replace:: keyboard shortcuts
+.. |Kbs| replace:: Keyboard shortcuts
+.. |os| replace:: operating system
+
+Users in Odoo can utilize several |kbs| to navigate through modules, execute actions, and manage
+data.
+
+.. image:: keyboard_shortcuts/menu-shortcuts.png
+   :align: center
+   :alt: A selection of keyboard shortcuts in Odoo.
+
+
+.. important::
+   Some |kbs| may not be effective for all users, depending on browsers, extensions, or other
+   individual settings.
+
+|Kbs| by |os|
+=============
+
+Below is a list of some of the most commonly used |kbs| within Odoo, listed by |os|.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Description
+     - Windows / Linux
+     - Mac
+   * - Previous breadcrumb
+     - :kbd:`Alt` + :kbd:`B`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`B`
+   * - Create new record
+     - :kbd:`Alt` + :kbd:`C`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`C`
+   * - Odoo Home Page
+     - :kbd:`Alt` + :kbd:`H`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`H`
+   * - Discard changes
+     - :kbd:`Alt` + :kbd:`J`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`J`
+   * - Save changes
+     - :kbd:`Alt` + :kbd:`S`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`S`
+   * - Change view
+     - :kbd:`Alt` + :kbd:`Shift` + :kbd:`V`
+     - :kbd:`Ctrl` + :kbd:`Shift` + :kbd:`V`
+   * - Pager next
+     - :kbd:`Alt` + :kbd:`N`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`N`
+   * - Pager previous
+     - :kbd:`Alt` + :kbd:`P`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`P`
+   * - Search
+     - :kbd:`Alt` + :kbd:`Q`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`Q`
+   * - Select menus
+     - :kbd:`Alt` + :kbd:`1-9`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`1-9`
+   * - Change company
+     - :kbd:`Alt` + :kbd:`Shift` + :kbd:`U`
+     - :kbd:`Ctrl` + :kbd:`Shift` + :kbd:`U`
+   * - Create a new To-Do
+     - :kbd:`Alt` + :kbd:`Shift` + :kbd:`T`
+     - :kbd:`Ctrl` + :kbd:`Shift` + :kbd:`T`
+   * - Search a Knowledge article
+     - :kbd:`Alt` + :kbd:`F`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`F`
+   * - Share a Knowledge article
+     - :kbd:`Alt` + :kbd:`Shift` + :kbd:`S`
+     - :kbd:`Ctrl` + :kbd:`Shift` + :kbd:`S`
+   * - Open command palette
+     - :kbd:`Ctrl` + :kbd:`K`
+     - :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`K`
+
+.. tip::
+  After opening the command palette, search using the following |kbs|:
+
+  - :kbd:`/`: search for menus, applications, and modules.
+  - :kbd:`@`: search for users.
+  - :kbd:`#`: search for **Discuss** channels.
+  - :kbd:`?`: search for **Knowledge** articles.
+
+  Enter a name or term in the search bar, or use the arrow keys to scroll through the available
```

Comment:
```suggestion
  Enter a name (or term) in the search bar, or use the arrow keys to scroll through the available
```
optional: added parenthetical here to provide a cleaner pacing through the sentence